docs/DeveloperGuide.md

```diff
@@ -342,11 +405,55 @@ We are also planning to shorten some input parameters when adding or editing a f
 | availability/nil          | avail/nil    |
 | animalType/               | type/        |
 
+With this, the command `add n/Jerry Tan p/98765412 e/jerry123@example.com housing/HDB avail/true` will be a valid add
+command. 
+
+### Reduce Coupling Between Availability and Animal Type
+Currently, the `animalType` field also contains information about the availability of a fosterer.<br>
+e.g. if the `animalType` field of a fosterer is `current.Dog`, it suggests that the fosterer is available, and is 
+fostering a dog. However, the user will still need to enter the `availability` field as `available`. <br>
```

Comment:
you mean "NotAvailable" here? since it is current.Dog

Reply:
sorry i meant able.Dog HAHAH so the fosterer is available! good catch